docker/Dockerfile-dhvirtualenv

```diff
@@ -50,17 +50,21 @@ FROM ${distro}
 ARG distro=""
 ENV distro ${distro}
 
+# Python < 3.7 assumes LANG="C" means ASCII-only and throws on printing unicode
+# http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
```

Comment:
this is the Dockerfile used to build the build environment, rather than the Dockerfile used to build the synapse debs. Are you sure this is the right place?

Reply:
I believe you're mistaken; this is a multi-stage Dockerfile which first sets up the build environment and then builds the debs in a separate stage. This ENV stanza is within the stage that actually does the building.

Reply:
oops. sorry.